### PR TITLE
integrate AirSwap widget

### PR DESCRIPTION
**Pull request by grahamperich**
This PR implements the AirSwap widget, which allows users to swap ETH for REP atomically with zero fees. Users just pay the gas cost of the transaction. 

I built Augur locally for development, but I also tested the integration by performing a swap on the Ethereum main net. I accomplished this by doing a little DOM hacking with the Chrome inspector. You can see how I tested it in this short screencast:

https://streamable.com/c99ay

Please let me know if there are any adjustments you'd like on this PR. We are big fans of Augur, and very excited for the main net launch! AirSwap also has some [developer resources](https://github.com/airswap/developers) if you're interested in further collaboration down the road.

### CODE REVIEW

File: src/modules/account/components/account-deposit/account-deposit.jsx [hunk 1]
```diff
@@ -48,6 +64,16 @@ export default class AccountDeposit extends Component {
           </button>
         </div>
       )
+      airSwapConverter = (
+        <div className={Styles.AccountDeposit__shapeShiftButton}>
+          <button
+            onClick={e => airSwapOnClick(e)}
+            value={'https://shapeshift.io/shifty.html?destination=' + address + '&output=REP'}
```
Comment (tinybike) [inline src/modules/account/components/account-deposit/account-deposit.jsx]: Hm, is this intended to be a ShapeShift link...?

Comment (grahamperich) [reply src/modules/account/components/account-deposit/account-deposit.jsx]: fixed, thank you.

**Comment by JohnDanz** [comment]
Is there any way to test this on Rinkeby? I'd rather not have to convert my real ETH/REP to test...

**Comment by grahamperich** [comment]
@JohnDanz I pushed a new commit which will point the widget to the airswap rinkeby sandbox when web3 is not set to main net. You can now test the functionality with testnet ETH.

**Pull request merged**